neurolang/probabilistic/dalvi_suciu_lift.py

```diff
@@ -163,6 +169,7 @@ def dalvi_suciu_lift(rule, symbol_table):
             for atom in extract_logic_atoms(rule)
         )
     ):
+        rule = MakeExistentialsImplicit().walk(rule)
```

Comment:
Just to take into account. This might actually miss the point that the existential variables must be projected out. If there is an existential present, maybe we should use a strategy similar to `separator_variable_plan` to project only the free variables in the expression. Maybe we should add an `extract_logic_bound_variables` which extracts all variables that are bound to an existential.